cibyl/models/ci/zuul/system.py

```diff
@@ -56,7 +56,6 @@ def __init__(self,
             top_level_model=Tenant,
             sources=sources,
             enabled=enabled,
```

Comment:
Tenants should be assigned to self.tenants in line 50

Reply:
Done